src/main/java/ua/danit/final_project/services/crud/VacancyServiceImpl.java

```diff
@@ -5,6 +5,7 @@
 import ua.danit.final_project.configuration.StaticCollection;
 import ua.danit.final_project.entities.User;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.entities.User'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=103&issues=AWW47gZy_eEZPtUmVxCI&open=AWW47gZy_eEZPtUmVxCI)

[ISSUE_KEY]: AWW47gZy_eEZPtUmVxCI